src/plugins/amqp10.js

```diff
@@ -66,7 +66,7 @@ function startReceiveSpan (tracer, config, link) {
     tags: {
       'resource.name': `receive ${source}`,
       'span.kind': 'consumer',
-      'amqp.link.source.address': source,
+      'message_bus.destination': source,
```

Comment:
Do you mind updating the `peer.*` tags as well?  They are currently `amqp.connection.*`

Reply:
They are already set elsewhere and tested here: https://github.com/signalfx/signalfx-nodejs-tracing/blob/959251116a316f33e14eccf656d395ecb195dabc/test/plugins/amqp10.spec.js#L75

So should I take out the `amqp.connection.*` tags?

Reply:
Since redundant, I'd think that's ok.

Reply:
I just looked through the instrumentation again, `peer.port` and `peer.hostname` are only set when sending messages not when consuming (and this is sufficient for inferred services). 

So the `amqp.connection.*` tag is only redundant when messages are being sent. Should I update it to `peer.*` to be in line with OT?